config.php.default

```diff
@@ -9,21 +9,16 @@ $config['db_pass'] = 'PASSWORD';
 $config['db_name'] = 'librenms';
 $config['db']['extension'] = 'mysqli';// mysql or mysqli
 
-### Memcached config - We use this to store realtime usage
-$config['memcached']['enable']  = FALSE;
-$config['memcached']['host']    = 'localhost';
-$config['memcached']['port']    = 11211;
-
 // This is the user LibreNMS will run as
 //Please ensure this user is created and has the correct permissions to your install
 $config['user'] = 'librenms';
 
 ### Locations - it is recommended to keep the default
-#$config['install_dir']  = "/opt/librenms";
+$config['install_dir']  = "/opt/librenms";
```

Comment:
The code I added should always be correct it checks the location of the file defaults.inc.php, then gets the parent dir.